src/cli.py

```diff
@@ -206,6 +228,26 @@ def main():
         minimizer.minimize(args.infile, args.outfile, args.num_inputs, args.add_fences)
         return
 
+    # Stand-alone generator
+    if args.subparser_name == "generate":
+        # if no seeds were given, select the ones from our config file
+        if not args.program_seed:
+            args.program_seed = CONF.program_generator_seed
```

Comment:
This line is redundant - in the end, all modules use CONF as the main source of configuration parameters. It's better to do it the other way around:
````
    if args.program_seed: 
        CONF.program_generator_seed = args.program_seed
   etc
````
and refactor `generate_test_batch` accordingly.

Let me take care of it, I'll push the change into your branch.